src/actions/myaction.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
```

Comment:
Could you rename this to a better verbose form?
`myAction.js` here would confuse a lot as we proceed further